business_calendar_test.go

```diff
@@ -337,3 +337,37 @@ func Test_businessCalendar_ConsistencyChecks(t *testing.T) {
 		}
 	})
 }
+
+func BenchmarkShiftBusinessDays(b *testing.B) {
```

Comment:
Not sure it's really necessary, but if for some reason the numbers used in here highlight some particular property keep it (maybe documented)

Reply:
That is @RemiMattheyDoret choices for the benchmarks, I do not really know which reasons lead him to choose these numbers...

Reply:
Ok, if that's the case we can keep this on hold and let him complete it :wink: or leave an issue for him, too - there's no pressure on this one